e2e/test/scenarios/permissions/sandboxes.cy.spec.js

```diff
@@ -505,257 +505,242 @@ describeEE("formatting > sandboxes", () => {
       });
     });
 
-    describe(
-      "with display values remapped to use a foreign key",
-      { tags: "@flaky" },
-      () => {
-        beforeEach(() => {
-          cy.intercept("POST", "/api/dataset").as("datasetQuery");
-          cy.log("Remap Product ID's display value to `title`");
-          remapDisplayValueToFK({
-            display_value: ORDERS.PRODUCT_ID,
-            name: "Product ID",
-            fk: PRODUCTS.TITLE,
-          });
+    describe("with display values remapped to use a foreign key", () => {
```

Comment:
The `@flaky` tag was applied to `describe`, so this PR stress-tests each test in this group.